DepositeCalcTests/Pages/CalculatorPage.cs

```diff
@@ -0,0 +1,210 @@
+                    return;
+                }
+
+                throw new Exception("Invalid financial year value");
+            }
+        }
+
+        public void ClickOnCalculateBtn()
+        {
+            CalculateBtn.Click();
+            new WebDriverWait(driver, TimeSpan.FromSeconds(2)).Until(_ => InterestEarnedFld.GetAttribute("value") != null);
+        }
+
+        public bool IsCalculateBtnDisabled => !CalculateBtn.Enabled;
+
+        public string InterestEarned => InterestEarnedFld.GetAttribute("value");
+
+        public string Income => IncomeFld.GetAttribute("value");
+
+        public string EndDateFldValue()
+        {
+            return EndDateFld.GetAttribute("value");
+        }
+
+        public void ClickOnDayDropDown()
+        {
+            DayDropDown.Click();
+        }
+        public void SendKeysToDayDropDown(string kay)
+        {
+            DayDropDown.SendKeys(kay);
+        }
+
+        public string GetDayStartDate()
+        {
+            string day = DayDropDown.GetAttribute("value");
+            return day;
+        }
+
+        public string GetDayEndDate()
+        {
+            string day = EndDateFld.GetAttribute("value");
+            return day.Substring(0, 2);
+        }
+
+        public int GetYearStartDate()
+        {
+            string year = YearDropDown.GetAttribute("value");
+            int yearBeforeCalculate = Convert.ToInt32(year);
+            return yearBeforeCalculate;
+        }
+
+        public int GetYearEndDate()
+        {
+            string year = EndDateFld.GetAttribute("value");
+            int yearAfterCalculate = Convert.ToInt32(year.Substring(6, 4));
+            return yearAfterCalculate;
+        }
+
+        public void ClickOnYearDropDown()
+        {
+            YearDropDown.Click();
+        }
+
+        public void ClickOnYear2024()
+        {
+            Year2024.Click();
+        }
+
+        public void ClickOnYear2028()
+        {
+            Year2028.Click();
+        }
+
+        public void SelectDay(string day)
+        {
+            var selectDayDropDown = DayDropDown;
+            var selectDayDropDownElement = new SelectElement(selectDayDropDown);
+            selectDayDropDownElement.SelectByText(day);
+        }
+
+        public void SelectMonth(string month)
+        {
+            var selectMonthDropDown = MonthDropDown;
+            var selectMonthDropDownElement = new SelectElement(selectMonthDropDown);
+            selectMonthDropDownElement.SelectByText(month);
+        }
+
+        public void SelectYear(string year)
+        {
+            var selectYearDropDown = YearDropDown;
+            var selectYearhDropDownElement = new SelectElement(selectYearDropDown);
+            selectYearhDropDownElement.SelectByText(year);
+        }
+
+        public string StartDateMonth
+        {
+            get
+            {
+                return MonthDropDown.GetAttribute("value");
+            }
```

Comment:
1. You can write it in one line;
2. You can use `SelectedOption` to get its text;
```suggestion
            get => new SelectElement(MonthDropDown).SelectedOption.Text;
```